src/source/PeerConnection/PeerConnection.c

```diff
@@ -39,20 +39,68 @@ STATUS allocateSrtp(PKvsPeerConnection pKvsPeerConnection)
     return retStatus;
 }
 
+typedef struct {
+    UINT32 currentDataChannelId;
+    PKvsPeerConnection pKvsPeerConnection;
+    PHashTable unkeyedDataChannels;
+} AllocateSctpSortDataChannelsData, *PAllocateSctpSortDataChannelsData;
+
+STATUS allocateSctpSortDataChannelsDataCallback(UINT64 customData, PHashEntry pHashEntry)
+{
+    STATUS retStatus = STATUS_SUCCESS;
```

Comment:
Do the inputs need sanitization? Have they been sanitized before the call?

Reply:
Yep!

Reply:
Yep!